### PR TITLE
Make alacritty(5) config paths resemble alacritty(1)

### PR DESCRIPTION
**Pull request by kchibisov**
--

I made them look like in `alacritty(1)`, though, maybe we should adjust both...

### CODE REVIEW

File: extra/man/alacritty.5.scd [hunk 1]
```diff
@@ -11,17 +11,15 @@ can be found at _https://toml.io/en/v1.0.0_.
 
 # LOCATION
 
-Alacritty doesn't create the config file for you, but it looks for one in the
-following locations on UNIX systems:
+Alacritty doesn't create the config file for you, but it looks for one at the
+following paths:
```
Comment (chrisduerr) [inline extra/man/alacritty.5.scd:15]: Why was this changed?

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: extra/man/alacritty.5.scd (hunk 1 above)
Comment (kchibisov) [reply extra/man/alacritty.5.scd:15]: to have the same wording as in `alacritty(1)`

Comment (chrisduerr) [reply extra/man/alacritty.5.scd:15]: Didn't we change this intentionally to explicitly call out unix? So shouldn't `alacritty(1)` be changed?

Comment (kchibisov) [reply extra/man/alacritty.5.scd:15]: yeah, one of them should be changed, so I'll change whatever you prefer.

Comment (chrisduerr) [reply extra/man/alacritty.5.scd:15]: I prefer changing it to explicitly point these out as unix paths. So I'd change `alacritty(1)`.

**Review by chrisduerr — APPROVED** [review]
